test/optimizations/fastFollowers/staticCheck.prediff

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+grep dynamicFastFollow c_code/staticCheck.c >> $1.exec.out.tmp
```

Comment:
Leftover debug code, or is this meant to count the number of dynamic checks in the generated code?

Reply:
This is my attempt to make sure that there is no dynamic check call in the generated code. Do you have a better approach in mind?

Reply:
Oh, duh -- gotcha. This makes sense, I was just confused by which file it was redirecting too. `$2` is the name of the logfile, so you should just be able to do `grep dynamicFastFollow c_code/staticCheck.c >> $2`

Longer term, I wonder if there's a more general way to do this that would work for llvm. The only thing that comes to mind is a compilerWarning or something in the slow follower, but that either means adding a test distribution (which seems like a lot of code) or introducing a param protected warning in block, which feels wonky.

Reply:
I have toyed with two different approaches that I believe can also work with LLVM:

- Hijack the distribution's follower in the test code, do a `writeln` forward to the actual follower. I implemented something similar to this [here](https://github.com/chapel-lang/chapel/blob/master/test/optimizations/autoLocalAccess/oneStaticFailOtherDynamicSuccess.chpl#L6-L14)
- Instead of grepping the generated code, grep something in the AST dump. Likely the dump of codegen.